protected/humhub/modules/admin/views/group/edit.php

```diff
@@ -20,15 +22,20 @@
             'form' => $form,
             'model' => $group,
             'attribute' => 'defaultSpaceGuid',
-            'selection' => $group->defaultSpace,
-            'maxSelection' => 1,
+            'selection' => $group->defaultSpaces,
+            'maxSelection' => 1000,
         ])
         ?>
     <?php endif; ?>
 
+    <div class="reassign-spaces">
+        <?= CHtml::a(Yii::t('AdminModule.user', 'Reassign All'), Url::toRoute(['/admin/group/reassign-all', 'id' => $group->id]), ['class' => 'btn btn-primary btn-reassign']); ?>
```

Comment:
We should migrate all `CHtml::` calls to `Html::`.

In general we should optimize the UI a little bit, but I'll give some separate feedback on that.

Reply:
Done!